apps/federatedfilesharing/lib/OCM/CloudFederationProviderFiles.php

```diff
@@ -335,13 +341,13 @@ private function mapShareTypeToNextcloud($shareType) {
 		return $result;
 	}
 
-	private function notifyAboutNewShare($shareWith, $shareId, $ownerFederatedId, $sharedByFederatedId, $name): void {
+	private function notifyAboutNewShare($shareWith, $shareId, $ownerFederatedId, $sharedByFederatedId, $name, $displayName): void {
```

Comment:
```suggestion
	private function notifyAboutNewShare($shareWith, $shareId, $ownerFederatedId, $sharedByFederatedId, $name, string $displayName): void {
```
And might as well type all parameters, this is a private function.